### ISSUE TITLE
Region based wiki top level pages are valid 'community' aspects

### ISSUE THREAD

**Issue by skylerbunny** [osmlab/osm-community-index#74]
(Example): Wisconsin Wiki mapping practices and ongoing projects (if you don't want to do this state-by-state, perhaps at least the national level Wikiproject page could be linked to: https://wiki.openstreetmap.org/wiki/Wisconsin

## Community Resource Name

Wiki suggested mapping guidelines and regional projects

### What is it?

Wiki page (and probably 'some' regional Wiki page exists for most places for which an OSM community does - this is particularly true in the US

### URL link to the resource

https://wiki.openstreetmap.org/wiki/Wisconsin

### Points of contact

If someone edits the Wiki and the page is watched, affected parties will be notified through the Wiki system

### Description (optional)

Lists projects and suggested standards used to map the state of Wisconsin.

### Languages spoken (optional)

en

### Country codes (optional)

us

### Add an emoji flag (optional)

us

**Comment by skylerbunny** [comment]
Why do this?

Many mappers aren't on mailing lists or doing meetups, but do frequently edit and watch their regional Wikiproject pages.

**Comment by bhousel** [comment]
This is an interesting suggestion.. I've always thought the wiki project pages are a bit more for serious mappers than for beginners, and I'm not sure whether sending a lot of new mappers at a wiki project page is good for either the beginners or for the project.  

However if a lot of people want this, I'll add it - let's discuss...

I think the _ideal_ solution would be if we get [OSM groups](https://github.com/osmlab/osm-planning/issues/1) implemented and then link to _that_ as a resource instead.  "Groups" is a very long-standing wish for OpenStreetMap that has evolved a bunch over the past 5 years.  I'm going to put some thoughts on that ticket.

**Comment by javiersanp** [comment]
I support this issue. At least in Spain, the national page have a lot of resources in Spanish of interest for new and non-English speakers mappers. Our community promotes the regional pages for local tips and activities.

**Comment by andrewharvey** [comment]
I'm semi for this idea. In Australia we have a wiki page at https://wiki.openstreetmap.org/wiki/Australian_Tagging_Guidelines. On one hand it's quite technical and detailed, but on the other hand good for beginners as it helps translate things as their are known locally to how they are mapped in OSM.

**Comment by skylerbunny** [comment]
An ongoing theme I've seen regarding the purpose of the column of links in iD as "for beginners" is that it isn't displayed "for beginners", even if a portion of its original intent (at [#4815)](https://github.com/openstreetmap/iD/issues/4815) is to give beginners some additional resources. The column is displayed for _all users_ regardless of number of edits or seniority. The column and links themselves are "new" for all editors that use iD as of v2.8.0.

The Wiki link doesn't have to appear first. My argument is that it _appear_, because an editor regardless of their total OSM experience may find that link of use to them. A beginner who isn't a beginner anymore after months of editing may still click any of the links for their region, including a Wiki link. If they get confused or want more direct help, they can and _should_ go right back to the other links in the list.

Last in a list would be just fine with me.

**Comment by skylerbunny** [comment]
_> it's also not really a community where you can ask for help or advice directly, just a good source of locally relevant information_

Since the links that appear vary based on the locale in which you are editing, this is exactly why I think inclusion of Wiki links makes sense. "A good source of locally relevant information" is a rather strong case.

**Comment by javiersanp** [comment]
I consider my self a senior editor, but I have very bad memory and as English is not my mother language I have to consult the Wiki all the time. The wiki is a resource that is not necessarily discovered quickly when you start and I think that links to pages written in the local language, with resources of interest locally are a valid and ussefull case use for this tool.

**Comment by treestryder** [comment]
I like the new feature, showing local information after an edit and kind of wish it was shown before and during an edit. I found this Issue, just before requesting to add my City's email list and State's Facebook page. However, much prefer linking to the regional Wiki pages. In this way, new users can learn more about the community they are mapping in. While the the local community only needs to keep their Wiki pages up to date.

Lately, I have been messaging new users and sharing the links to the wiki pages and online communities, but something like the following line would be far better (and I imagine, easier for the iD team to maintain)...

[Lansing](https://wiki.openstreetmap.org/wiki/Lansing), [Michigan](https://wiki.openstreetmap.org/wiki/Michigan), [United States](https://wiki.openstreetmap.org/wiki/United_States)

**Comment by harry-wood** [comment]
I agree that wiki pages are sometimes a good "resource" related to a city/region. Sometimes the _main_ resource, for example I always used the [London wiki page](https://wiki.openstreetmap.org/wiki/London) as the main "website" of the OSMLondon community, including updating events lists on there. Hopefully it is reasonably welcoming to beginners. So I support this suggestion, at least as an option where it's useful.

It's probably a bit unusual to use a city/region wiki page in that way. There's two other things which happen on a wiki page. Either it doesn't get updated much at all, or it gets regularly edited by mappers busily maintaining mapping status information. In both cases (whether you're using your wiki page, or leaving it largely static) I would like to encourage people to put helpful information for beginners prominently near the top. It shouldn't really be that _"wiki project pages are a bit more for serious mappers than for beginners"_ but in practice, I know what you mean.

Funny thing is, the main "helpful information for beginners" which I've encouraged people to stick on their wiki page is... a set of links to facebook groups, meetup.com, mailing lists, or whatever channels/places where this community get together. Sound familiar? :-)  I never encouraged people very loudly, or went on a wiki editing rampage, but maybe some time soon I'll be able to do so, copying a bunch of links out of this repo. In fact I imagine with a bit of scripting we might be able to scan for inconsistencies between the community links on osm-community-index<->wiki.
